src/main/java/com/netflix/dgs/plugin/services/internal/GraphQLSchemaRegistry.java

```diff
@@ -37,11 +38,12 @@ public GraphQLSchemaRegistry(Project project) {
 
     public @Nullable
     Optional<PsiElement> psiForSchemaType(@NotNull PsiElement psiElement, @Nullable String parentType, @Nullable String field) {
-
         TypeDefinitionRegistry registry = getRegistry(psiElement);
-        Optional<ObjectTypeDefinition> objectType = getTypeDefinition(registry, parentType);
-        if (objectType.isPresent()) {
-            Optional<FieldDefinition> schemaField = objectType.get().getFieldDefinitions().stream().filter(f -> f.getName().equals(field)).findAny();
+        List<ObjectTypeDefinition> objectTypes = getTypeDefinitions(registry, parentType);
+        if (!objectTypes.isEmpty()) {
+            var fieldDefinitionList = new ArrayList<FieldDefinition>();
+            objectTypes.stream().map(ObjectTypeDefinition::getFieldDefinitions).forEach(fieldDefinitionList::addAll);
```

Comment:
Nit: do we need the array list above? Can't we just collect to list from the stream?

Reply:
you're right,stream is enough